docs/resources.md

```diff
@@ -172,8 +172,8 @@ cluster. The kubeconfig will be placed in
 
 The Cluster resource has the following parameters:
 
-- Name: The name of the Resource is also given to cluster, will be used in the
-  kubeconfig and also as part of the path to the kubeconfig file
+- Name (required): The name to be given to the target cluster, will be used 
```

Comment:
If `Name` is required then can it adds value to add this check in resource validation.

Reply:
added validation